Mods/Alpakit/Source/Alpakit/Private/Alpakit.cpp

```diff
@@ -135,11 +135,13 @@ TSharedRef<FAlpakitProfile> MakeDevelopmentProfileForMod(TSharedRef<IPlugin> Mod
 
     Profile->BuildConfiguration = Settings->GetBuildConfiguration();
 
-    for (auto [CookedPlatform, TargetSetting] : Settings->GetPlatformTargetSettings()) {
+    for (auto& [CookedPlatform, TargetSetting] : Settings->GetPlatformTargetSettings()) {
         if (TargetSetting.bEnabled) {
             Profile->CookedPlatforms.Add(CookedPlatform);
+            FAlpakitProfileGameInfo& GameInfo = Profile->PlatformGameInfo.FindOrAdd(CookedPlatform);
+            GameInfo.CustomLaunchPath = TargetSetting.CustomLaunchPath;
```

Comment:
Because each UAT task is ran sequentially, only the last task should launch the game, so this should go here https://github.com/satisfactorymodding/SatisfactoryModLoader/blob/master/Mods/Alpakit/Source/Alpakit/Private/Alpakit.cpp#L165-L175